weather/weatherapp/views.py

```diff
@@ -3,17 +3,52 @@
 import requests
 from .models import City
 from .forms import CityForm
+import os 
+
+
+apiId = os.environ['weatherapp_api_key']
 
 def index(request):
-    url = 'http://api.openweathermap.org/data/2.5/weather?q={}&units=imperial&appid= '
+    url = 'http://api.openweathermap.org/data/2.5/weather?q={}&units=imperial&appid=%s' %apiId
```

Comment:
you are adding **apiID** at the end of the URL string, right? 
not sure about the syntax but it looks like something is missing. Does it work?